pkg/util/util.go

```diff
@@ -147,3 +149,106 @@ func GetFileChunkChecksum(filePath string, start, size int64) (string, error) {
 
 	return hex.EncodeToString(h.Sum(nil)), nil
 }
+func ParseLabels(labels []string) (map[string]string, error) {
```

Comment:
todo: Could consider moving utils functions here to go-common-libs if appropriate later.